Library/Homebrew/.rubocop_todo.yml

```diff
@@ -60,19 +60,26 @@ Lint/ShadowedException:
     - 'utils/fork.rb'
 
 # Offense count: 13
+# Configuration parameters: CountBlocks.
 Metrics/BlockNesting:
   Max: 5
 
-# Offense count: 19
+# Offense count: 21
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 400
+  Max: 364
```

Comment:
Let's leave this at `400` just so it doesn't 💥 if you need to add a line to e.g. `bottle.rb`